doc/src/modules/vector/vector_integration.rst

```diff
@@ -0,0 +1,144 @@
+================================
+Applications of Vector Integrals
+================================
+
+To integrate a scalar or vector field over a region, we have to first define a region. SymPy provides three methods for defining a region:
+
+1. Using Parametric Equations.
+
+2. Using Implicit Equation.
+
+3. Using objects of geometry module.
+
+The :func:`~sympy.vector.integrals.vector_integrate` function is used to integrate scalar or vector field over any type of region. It automatically determines the type of integration (line, surface or volume) depending on the nature of the object.
+
+We define a coordinate system and make necesssary imports for examples.
+>>> from sympy import sin, cos, exp, pi, symbols
+>>> from sympy.vector import CoordSys3D, ParametricRegion, ImplicitRegion, vector_integrate
+>>> from sympy.abc import r, x, y, z, theta, phi
+>>> C = CoordSys3D('C')
+    
+Calculation of Perimeter, Surface Area and Volume
+=================================================
+
+To calculate the perimeter of circle, we need to define a circle. Let's define it using its parametric equation.
+
+>>> param_circle = ParametricRegion((4*cos(theta), 4*sin(theta)), (theta, 0, 2*pi))
+
+We can also define circle using its implicit equation.
+
+>>> implicit_circle = ImplicitRegion((x, y), x**2 + y**2 - 4)
+
+Perimeter of a figure is equal to the absolute value of its integral over a unit scalar field. 
+
+>>> vector_integrate(1, param_circle)
+8*pi
+>>> vector_integrate(1, implicit_circle)
+8*pi
+
+Suppose a user wants to calculate the perimeter of triangle. Determining the parametric representation of triangle can be difficult. Instead, the user can use an object of :class:`~sympy.geometry.polygon.Polygon` class in the geometry module.
+
+>>> from sympy.geometry import Point, Polygon 
+>>> triangle = Polygon(Point(1, 2), (3, 5), (1,6))
+>>> vector_integrate(1, triangle)
+sqrt(5) + sqrt(13) + 4
+
+To define a solid sphere, we need to use three parameters (r, theta and phi). 
+
+>>> solidsphere = ParametricRegion((r*sin(phi)*cos(theta),r*sin(phi)*sin(theta), r*cos(phi)),
+... (phi, 0, pi), (theta, 0, 2*pi), (r, 0, 3))
+>>> abs(vector_integrate(1, solidsphere))
+36*pi
+
+Calculation of mass of a body
+=============================
+
+Consider a triangular lamina 𝑅  with vertices (0,0), (0, 5), (5,0) and with density :math:`\rho(x, y) = xy\:kg/m^2`. Find the total mass.
+
+>>> triangle = ParametricRegion((x, y), (x, 0, 5), (y, 0, 5 - x))
```

Comment:
does this work with the `polygon` object as well?

Reply:
No. `vector_integrate` will perform line integral over the sides of the polygon instead of calculating the area enclosed.

Reply:
OK, maybe that's an issue for some future work.

Reply:
Yes. We can add new classes in geometry module for representing such regions.

Reply:
I will open an issue for this then.

Reply:
It would require an algorithm to decompose the polygon into triangles (or pyramids in the 3D case). Well, maybe for GSoC 2021 :)

Reply:
Yes, for polygons it will be difficult. But I think adding a class to represent a triangular area will be easier.